client/src/app/site/agenda/services/agenda-repository.service.ts

```diff
@@ -45,6 +54,87 @@ export class AgendaRepositoryService extends BaseRepository<ViewItem, Item> {
         }
     }
 
+    /**
+     * Generate viewSpeaker objects from a given agenda Item
+     * @param item agenda Item holding speakers
```

Comment:
proper `@returns` is missing in the documentation

Reply:
yep